rust/cloud-storage/search_service/src/api/search/project.rs

```diff
@@ -33,7 +34,7 @@ pub(in crate::api::search) async fn enrich_projects(
         return Ok(vec![]);
     }
     // Extract project IDs from results
-    let project_ids: Vec<String> = results.iter().map(|r| r.entity_id.clone()).collect();
+    let project_ids: Vec<String> = results.iter().map(|r| r.entity_id.to_string()).collect();
```

Comment:
change to `Vec<Uuid>`?

Reply:
that requires changing a lot more since macro_db_client expects string still for those ids. i'd like to keep those changes separate as they are in a completely separate service